### PR TITLE
cc 11

### PR DESCRIPTION
**Pull request by ZenAtomsk**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: javascript/code-challenges/queueWithStacks/queue-with-stacks.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+      for(let i = 0; i < bLength; i++){
+        let temp = stackB.pop();
+        this.stackA.push(temp);
+      }
+    }
+    this.stackA.push(value);
+  }
+
+  // dequeue()
```
Comment (adrianhuebner) [inline javascript/code-challenges/queueWithStacks/queue-with-stacks.js:47]: Everything is looks really great Ricardo! Would just like to see some tests added!
